ra2ce/analyses/indirect/traffic_analysis/equity_analysis.py

```diff
@@ -0,0 +1,100 @@
+"""
+                    GNU GENERAL PUBLIC LICENSE
+                      Version 3, 29 June 2007
+
+    Risk Assessment and Adaptation for Critical Infrastructure (RA2CE).
+    Copyright (C) 2023 Stichting Deltares
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import geopandas as gpd
+import pandas as pd
+import numpy as np
+from ra2ce.analyses.indirect.traffic_analysis.accumulated_traffic_dataclass import (
+    AccumulatedTraffic,
+)
+from ra2ce.analyses.indirect.traffic_analysis.traffic_analysis_base import (
+    TrafficAnalysisBase,
+)
+
+
+class EquityAnalysis(TrafficAnalysisBase):
+    """
+    Specialization from `TrafficAnalysisBase` which takes into account the equity data (weights per region).
+    """
+
+    gdf: gpd.GeoDataFrame
+    od_table: gpd.GeoDataFrame
+    equity_data: pd.DataFrame
+    destinations_names: str
+
```

Comment:
I get that we need an od_table, equity_data but hat is this gdf?

Reply:
`gdf` is the acronym of `GeoDataFrame`, it actualy refers to the "main" dataframe containing all the road network information, but for the moment I don't dare changing its name as I'm not entirely sure (we are using this name already in the `networks.py` file.